bot/exts/code_jams/_cog.py

```diff
@@ -185,6 +185,66 @@ async def remove(self, ctx: commands.Context, member: Member) -> None:
             view=JamConfirmation(author=ctx.author, callback=callback)
         )
 
+    @codejam.command()
+    @commands.has_any_role(Roles.admins, Roles.events_lead, Roles.code_jam_event_team, Roles.code_jam_participants)
+    @in_code_jam_category(_creation_utils.CATEGORY_NAME)
+    async def pin(self, ctx: commands.Context) -> None:
+        """Lets Code Jam Participants to pin messages in their team channels."""
+        referenced_message = getattr(ctx.message.reference, "resolved", None)
```

Comment:
Can we add the ability to check if they're passing in a message link? Not sure how much it would be used but it would be nice to have the option.

We would have to check if the message exists in the current channel though.

Reply:
Sure, we can.